### DISCUSSION TITLE
The other version

### DISCUSSION THREAD

**Discussion by Lastofthefirst** [nocodb/nocodb#791] [Discord Threads]
I'm having a hard time reconciling these two images/messages, one is a screenshot from the website, the other is a screenshot from the dashboard. Is it playful banter? `¯\_(ツ)_/¯`

<img width="1183" alt="Screen Shot 2021-11-17 at 7 08 16 PM" src="https://user-images.githubusercontent.com/47003973/142302159-48bde35c-f41d-4838-a527-fb60aa5537e4.png">

<img width="838" alt="Screen Shot 2021-11-17 at 7 08 41 PM" src="https://user-images.githubusercontent.com/47003973/142302145-19639737-926f-46f5-a97e-3c2e1fc40099.png">

**Comment by Lastofthefirst** [comment]
Oof, I see this in the docs, that's not promising.

<img width="924" alt="Screen Shot 2021-11-17 at 7 32 18 PM" src="https://user-images.githubusercontent.com/47003973/142314448-2c37bcf8-9d8e-4464-896b-e45302e72bb6.png">

**Comment by o1lab** [comment]
Thank you for highlighting this.

The difference is cause :
- We got the page from our previous API product which was a precursor into NocoDB.  
- Hence much of the questions revolved around API usage and product.  

For months now, we are just focused on building the product and had no time to update the website & were not keeping track of what part of that was duplicated in docs (should very well point out website to github now .. lol) 

To answer, we just have one version of Noco that is free & open source. Most of the premium/pro features that you find in other products are **just freely available in NocoDB**. And we do not intend to move them to enterprise edition in future. 

Do we plan to have enterprise edition ? For features that make sense for enterprises such as SSO, SLA, Org wide reports, advanced audit trails etc - yes.
